Error when running latest version

### Description

```
jraad@MacBookPro /Users/jraad $ npm install -g @pensar/apex
  npm warn deprecated tar@6.2.1: Old versions of tar are not supported, and contain widely
  publicized security vulnerabilities, which have been fixed in the current version. Please
  update. Support for old versions may be purchased (at exorbitant rates) by contacting
  i@izs.me

  added 12 packages, removed 20 packages, and changed 371 packages in 14s

  56 packages are looking for funding
    run `npm fund` for details
  jraad@MacBookPro /Users/jraad $ pensar
  error: Cannot find module '../src/core/installation/index.ts' from
  '/Users/jraad/.nvm/versions/node/v22.15.0/lib/node_modules/@pensar/apex/bin/pensar.js'

  Bun v1.2.18 (macOS arm64)
  jraad@MacBookPro /Users/jraad $ pensar --version
  error: Cannot find module '../src/core/installation/index.ts' from
  '/Users/jraad/.nvm/versions/node/v22.15.0/lib/node_modules/@pensar/apex/bin/pensar.js'

  Bun v1.2.18 (macOS arm64)


```

### Apex version

_No response_

### AI Provider

_No response_

### Steps to reproduce

_No response_

### Screenshots or logs

_No response_

### Operating System

_No response_

### Terminal

_No response_

Having this issue with npm and bun.
```
jraad@MacBookPro /Users/jraad/projects/apex $  npm uninstall -g @pensar/apex


removed 383 packages in 2s
jraad@MacBookPro /Users/jraad/projects/apex $ bun install -g @pensar/apex

bun add v1.2.18 (0d4089ea)

installed @pensar/apex@0.0.70 with binaries:
 - pensar

369 packages installed [5.81s]
jraad@MacBookPro /Users/jraad/projects/apex $ pensar
error: Cannot find module '../src/core/installation/index.ts' from '/Users/jraad/.bun/install/global/node_modules/@pensar/apex/bin/pensar.js'

Bun v1.2.18 (macOS arm64)

```
@joshkotrous

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -27,3 +27,9 @@ jobs:
 
       - name: Build
         run: bun run build
+
+      - name: Smoke test published package
+        run: |
+          npm pack
+          npm install -g ./pensar-apex-*.tgz
+          pensar --version

--- a/package.json
+++ b/package.json
@@ -15,6 +15,7 @@
   "files": [
     "build",
     "bin",
+    "src/core/installation",
     "pensar.svg",
     "LICENSE"
   ],

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@pensar/apex",
-  "version": "0.0.70",
+  "version": "0.0.71",
   "description": "AI-powered penetration testing CLI tool with terminal UI",
   "module": "src/tui/index.tsx",
   "main": "build/index.js",

